servicebroker/config/settings.json

```diff
@@ -1,5 +1,7 @@
 {
   "port": 8080,
   "user": "user",
```

Comment:
You should pull in the latest develop and rebase your branch. The rebase will probably have a conflict which should force you to fix things. At which point once you commit the rebase you should be good with a PR.